packages/docusaurus-plugin-client-redirects/src/createRedirectPageContent.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+type CreateRedirectPageOptions = {
+  toUrl: string;
+};
+
+export default function createRedirectPageContent({
+  toUrl,
+}: CreateRedirectPageOptions) {
+  return `
+<!DOCTYPE html>
```

Comment:
How about using Eta to render an HTML template from separated file? You can find an example below: https://github.com/facebook/docusaurus/blob/368eb4279ff6a1f26d134a2349e2ebd8d9c35644/packages/docusaurus/src/client/serverEntry.js#L64

Reply:
done